src/index.ts

```diff
@@ -126,7 +125,11 @@ function initializeProvider(providerOrRpcUrl, gasInfo) {
 
   // if provider is a string, assume it's a rpcUrl
   if (typeof providerOrRpcUrl === 'string') {
-    return new ProviderWrapper(new HttpProvider(providerOrRpcUrl), gasInfo);
+    const client = createPublicClient({
+      chain: lukso,
+      transport: http('https://rpc.testnet.lukso.network'),
```

Comment:
should we be hardcoding the rpc url here? and is it going to be always testnet?

Reply:
Ah good catch, thanks.